nin/backend/blank-project/index.html

```diff
@@ -0,0 +1,130 @@
+<html>
+<head>
+  <meta charset=utf8>
+  <title>My project</title>
+  <style>
+    * {
+      font-family: sans-serif;
+      display: none;
+      color: transparent;
+    }
+
+    .visible, .visible * {
+      display: block;
+    }
+
+    html, body {
+      width: 100%;
+      height: 100%;
+      display: block;
+      overflow: hidden;
+      padding: 0;
+      margin: 0;
+      border: 0;
+      outline: 0;
+      background: -webkit-linear-gradient(top, #FFFFFF, #CCDDFF);
+      background: -moz-linear-gradient(top, #FFFFFF, #CCDDFF);
+      background-size: cover;
+    }
+
+    h1, h2 {
+      text-align: center;
+      color: black;
+    }
+
+    canvas {
+      position: fixed;
+      display: block;
+      background: #000;
+    }
+
+    .hide {
+      display: none;
+    }
+
+    .loading-text {
+      position: fixed;
+      left: 45%;
+      top: 52%;
+      width: 10%;
+      color: black;
+      text-align: center;
+      font-size: 14px;
+    }
+
+    .progress-bar-inner {
+      width: 0;
+      height: 100%;
+      background: #2c7f61;
+    }
+
+    .progress-bar-container {
+      position: fixed;
+      left: 45%;
+      top: 54.5%;
+      bottom: 43.5%;
+      width: 10%;
+      border: 1px solid #2c7f61;
+    }
+
+    #start-button {
+      position: fixed;
+      left: 45%;
+      top: 54.5%;
+      width: 10%;
+      color: black;
+      text-decoration: none;
+      text-align: center;
+      border: 1px solid black;
+      padding: 5px;
+      margin-top: -10px;
+    }
+    #pouet-link {
+      position: fixed;
+      bottom: 10px;
+      text-align: center;
+      color: gray;
+      left: 0;
+      right: 0;
+      font-size: 14px;
+    }
+  </style>
+</head>
+
+<body>
+  <div id=wrapper class=visible>
+    <div class=loading-text>Loading...</div>
+    <div class=progress-bar-container>
+      <div class=progress-bar-inner>
+      </div>
+    </div>
+    <a id=start-button class=hide href=javascript:; onclick=STARTDEMO();>
+      START
+    </a>
+    <a href="http://www.pouet.net/groups.php?which=11702" target="_blank" id="pouet-link">Read more about this demo on Pouet</a>
+  </div>
+  <script>
+  var loadingText = document.querySelector('.loading-text');
+  var progressBarInner = document.querySelector('.progress-bar-inner');
+  var progressBarContainer = document.querySelector('.progress-bar-container');
+  var wrapper = document.querySelector('#wrapper');
+  var startButton = document.querySelector('#start-button');
+  ONPROGRESS = function(progress) {
+    progressBarInner.style.width = ((+progress * 100 | 0) / 100) + '%';
+  };
+
+  ONCOMPLETE = function() {
+    progressBarInner.style.width = '100%';
+    loadingText.classList.add('hide');
+    progressBarContainer.classList.add('hide');
+    startButton.classList.remove('hide');
+  };
+
+  STARTDEMO = function() {
+    wrapper.classList.remove('visible');
+    wrapper.style.background = '#fff';
+    document.body.style.background = '#000';
+    demo.start();
```

Comment:
Would be a good idea in my opinion. Ended up doing this on Crankwork in order to let the button change name to `Starting..!`, since the warmup does take a couple of seconds.

You can change this function to something like this (taken directly from Crankwork):
```
STARTDEMO = function() {
  startButton.innerHTML = 'Starting..!';
  setTimeout(function() {
    wrapper.classList.remove('visible');
    wrapper.style.background = '#fff';
    document.body.style.background = '#000';
    demo.start();
  }, 100);
};
```